### ISSUE TITLE
Calico updates to iptables

### ISSUE THREAD

**Issue by akasetty** [apprenda/kismatic#567]
Using Kismatic 1.3.3 version,  installed K8s 1.6.4 on Centos 7.3.  Everything works fine but I keep seeing the following logs from Calico nodes

Is this normal,  the updates to IPtables keeps going on for ever.  
```
017-06-02 15:21:04.786 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="raw"
2017-06-02 15:21:04.786 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="raw"
2017-06-02 15:21:04.789 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.885041
2017-06-02 15:21:07.177 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:07.178 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="nat"
2017-06-02 15:21:07.178 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="nat"
2017-06-02 15:21:07.184 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=6.799722
2017-06-02 15:21:08.057 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:08.057 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x4
2017-06-02 15:21:08.057 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x6
2017-06-02 15:21:08.057 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet"
2017-06-02 15:21:08.057 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet6"
2017-06-02 15:21:08.058 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet"
2017-06-02 15:21:08.059 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet6"
2017-06-02 15:21:08.070 [INFO][82] ipsets.go 283: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=12.468599ms
2017-06-02 15:21:08.062 [INFO][82] ipsets.go 283: Finished resync family="inet6" numInconsistenciesFound=0 resyncDuration=2.48551ms
2017-06-02 15:21:08.071 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=13.338612
2017-06-02 15:21:08.391 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:08.392 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="filter"
2017-06-02 15:21:08.392 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="filter"
2017-06-02 15:21:08.394 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=2.696147
2017-06-02 15:21:09.578 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:09.578 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="raw"
2017-06-02 15:21:09.578 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="raw"
2017-06-02 15:21:09.580 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="nat"
2017-06-02 15:21:09.580 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="nat"
2017-06-02 15:21:09.592 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="filter"
2017-06-02 15:21:09.592 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="filter"
2017-06-02 15:21:09.595 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=16.239063
2017-06-02 15:21:14.787 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:14.788 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="raw"
2017-06-02 15:21:14.788 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="raw"
2017-06-02 15:21:14.816 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=28.507889000000002
2017-06-02 15:21:17.210 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:17.214 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="nat"
2017-06-02 15:21:17.214 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="nat"
2017-06-02 15:21:17.291 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=77.75729100000001
2017-06-02 15:21:18.464 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:18.464 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="filter"
2017-06-02 15:21:18.464 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="filter"
2017-06-02 15:21:18.490 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=25.657268000000002
2017-06-02 15:21:18.783 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:18.784 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x4
2017-06-02 15:21:18.784 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x6
2017-06-02 15:21:18.784 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet"
2017-06-02 15:21:18.784 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet6"
2017-06-02 15:21:18.795 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet"
2017-06-02 15:21:18.811 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet6"
2017-06-02 15:21:18.818 [INFO][82] ipsets.go 283: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=22.250972ms
2017-06-02 15:21:18.819 [INFO][82] ipsets.go 283: Finished resync family="inet6" numInconsistenciesFound=0 resyncDuration=7.340383ms
2017-06-02 15:21:18.819 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=35.382892999999996
2017-06-02 15:21:19.583 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:19.593 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="raw"
2017-06-02 15:21:19.593 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="raw"
2017-06-02 15:21:19.617 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="nat"
2017-06-02 15:21:19.617 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="nat"
2017-06-02 15:21:19.640 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="filter"
2017-06-02 15:21:19.640 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="filter"
2017-06-02 15:21:19.696 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=103.64933599999999
2017-06-02 15:21:24.851 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:24.858 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="raw"
2017-06-02 15:21:24.858 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="raw"
2017-06-02 15:21:24.869 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=11.288509000000001
2017-06-02 15:21:27.226 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:27.226 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="nat"
2017-06-02 15:21:27.226 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="nat"
2017-06-02 15:21:27.236 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=9.629712
2017-06-02 15:21:28.478 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:28.482 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="filter"
2017-06-02 15:21:28.482 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="filter"
2017-06-02 15:21:28.487 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=8.284987000000001
2017-06-02 15:21:28.893 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:28.897 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x4
2017-06-02 15:21:28.897 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x6
2017-06-02 15:21:28.897 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet"
2017-06-02 15:21:28.897 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet6"
2017-06-02 15:21:28.912 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet"
2017-06-02 15:21:28.921 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet6"
2017-06-02 15:21:28.938 [INFO][82] ipsets.go 283: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=25.887124ms
2017-06-02 15:21:28.938 [INFO][82] ipsets.go 283: Finished resync family="inet6" numInconsistenciesFound=0 resyncDuration=17.317662ms
2017-06-02 15:21:28.938 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=45.126422999999996
2017-06-02 15:21:29.596 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:29.601 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="raw"
2017-06-02 15:21:29.602 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="raw"
2017-06-02 15:21:29.607 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=5.276696
2017-06-02 15:21:29.619 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:29.619 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="nat"
2017-06-02 15:21:29.619 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="nat"
2017-06-02 15:21:29.622 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.029885
2017-06-02 15:21:29.642 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:29.643 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="filter"
2017-06-02 15:21:29.643 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="filter"
2017-06-02 15:21:29.668 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=24.921665
2017-06-02 15:21:34.879 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:34.883 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="raw"
2017-06-02 15:21:34.883 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="raw"
2017-06-02 15:21:34.895 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=15.460932
2017-06-02 15:21:37.238 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:37.239 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="nat"
2017-06-02 15:21:37.239 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="nat"
2017-06-02 15:21:37.252 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=13.443421
2017-06-02 15:21:38.492 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:38.492 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="filter"
2017-06-02 15:21:38.492 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="filter"
2017-06-02 15:21:38.496 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=4.330054
2017-06-02 15:21:39.386 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:39.389 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x4
2017-06-02 15:21:39.389 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x6
2017-06-02 15:21:39.389 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet"
2017-06-02 15:21:39.390 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet6"
2017-06-02 15:21:39.391 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet"
2017-06-02 15:21:39.392 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet6"
2017-06-02 15:21:39.397 [INFO][82] ipsets.go 283: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=6.185675ms
2017-06-02 15:21:39.398 [INFO][82] ipsets.go 283: Finished resync family="inet6" numInconsistenciesFound=0 resyncDuration=5.415586ms
2017-06-02 15:21:39.398 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=11.269687000000001
2017-06-02 15:21:39.602 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:39.602 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="raw"
2017-06-02 15:21:39.602 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="raw"
2017-06-02 15:21:39.613 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=11.051593
2017-06-02 15:21:39.629 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:39.629 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="nat"
2017-06-02 15:21:39.629 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="nat"
2017-06-02 15:21:39.633 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=4.285837
2017-06-02 15:21:39.645 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:39.646 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="filter"
2017-06-02 15:21:39.646 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="filter"
2017-06-02 15:21:39.649 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.949701
2017-06-02 15:21:44.884 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:44.885 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="raw"
2017-06-02 15:21:44.885 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="raw"
2017-06-02 15:21:44.888 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.677023
2017-06-02 15:21:47.243 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:47.243 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="nat"
2017-06-02 15:21:47.243 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="nat"
2017-06-02 15:21:47.257 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=14.728341
2017-06-02 15:21:48.502 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:48.503 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="filter"
2017-06-02 15:21:48.503 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="filter"
2017-06-02 15:21:48.506 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.015687
2017-06-02 15:21:49.605 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:49.606 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="raw"
2017-06-02 15:21:49.606 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="raw"
2017-06-02 15:21:49.609 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.989525
2017-06-02 15:21:49.611 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:49.611 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x4
2017-06-02 15:21:49.611 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x6
2017-06-02 15:21:49.611 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet"
2017-06-02 15:21:49.611 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet6"
2017-06-02 15:21:49.611 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet"
2017-06-02 15:21:49.613 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet6"
2017-06-02 15:21:49.615 [INFO][82] ipsets.go 283: Finished resync family="inet6" numInconsistenciesFound=0 resyncDuration=1.012486ms
2017-06-02 15:21:49.615 [INFO][82] ipsets.go 283: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=4.004259ms
2017-06-02 15:21:49.616 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=4.921405
2017-06-02 15:21:49.629 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:49.629 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="nat"
2017-06-02 15:21:49.629 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="nat"
2017-06-02 15:21:49.632 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=2.37257
2017-06-02 15:21:49.646 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:49.647 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="filter"
2017-06-02 15:21:49.647 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="filter"
2017-06-02 15:21:49.650 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.7230950000000003
2017-06-02 15:21:54.886 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:54.886 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="raw"
2017-06-02 15:21:54.886 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="raw"
2017-06-02 15:21:54.907 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=20.544898999999997
2017-06-02 15:21:57.264 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:57.264 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="nat"
2017-06-02 15:21:57.264 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="nat"
2017-06-02 15:21:57.281 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=16.844017
2017-06-02 15:21:58.507 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:58.508 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="filter"
2017-06-02 15:21:58.508 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="filter"
2017-06-02 15:21:58.515 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=7.8134689999999996
2017-06-02 15:21:59.613 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:59.614 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="raw"
2017-06-02 15:21:59.614 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="raw"
2017-06-02 15:21:59.617 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.232563
2017-06-02 15:21:59.630 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:59.630 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="nat"
2017-06-02 15:21:59.630 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="nat"
2017-06-02 15:21:59.633 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=2.716834
2017-06-02 15:21:59.647 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:21:59.647 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="filter"
2017-06-02 15:21:59.648 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="filter"
2017-06-02 15:21:59.651 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.699584
2017-06-02 15:22:00.207 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:00.208 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x4
2017-06-02 15:22:00.208 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x6
2017-06-02 15:22:00.208 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet"
2017-06-02 15:22:00.208 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet6"
2017-06-02 15:22:00.208 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet"
2017-06-02 15:22:00.209 [INFO][82] ipsets.go 283: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=1.279577ms
2017-06-02 15:22:00.210 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet6"
2017-06-02 15:22:00.210 [INFO][82] ipsets.go 283: Finished resync family="inet6" numInconsistenciesFound=0 resyncDuration=891.536µs
2017-06-02 15:22:00.211 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.612043
2017-06-02 15:22:04.887 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:04.887 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="raw"
2017-06-02 15:22:04.887 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="raw"
2017-06-02 15:22:04.894 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=7.521249
2017-06-02 15:22:07.272 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:07.272 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="nat"
2017-06-02 15:22:07.272 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="nat"
2017-06-02 15:22:07.277 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=5.274694
2017-06-02 15:22:08.510 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:08.510 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="filter"
2017-06-02 15:22:08.510 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="filter"
2017-06-02 15:22:08.521 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=11.354669
2017-06-02 15:22:09.625 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:09.625 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="raw"
2017-06-02 15:22:09.625 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="raw"
2017-06-02 15:22:09.627 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=2.0752800000000002
2017-06-02 15:22:09.631 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:09.631 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="nat"
2017-06-02 15:22:09.631 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="nat"
2017-06-02 15:22:09.632 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=1.260543
2017-06-02 15:22:09.648 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:09.648 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="filter"
2017-06-02 15:22:09.648 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="filter"
2017-06-02 15:22:09.661 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=13.434157
2017-06-02 15:22:10.539 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:10.539 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x4
2017-06-02 15:22:10.539 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x6
2017-06-02 15:22:10.539 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet"
2017-06-02 15:22:10.539 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet6"
2017-06-02 15:22:10.540 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet"
2017-06-02 15:22:10.541 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet6"
2017-06-02 15:22:10.542 [INFO][82] ipsets.go 283: Finished resync family="inet6" numInconsistenciesFound=0 resyncDuration=823.889µs
2017-06-02 15:22:10.542 [INFO][82] ipsets.go 283: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=2.686952ms
2017-06-02 15:22:10.542 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.488692
2017-06-02 15:22:14.887 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:14.887 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="raw"
2017-06-02 15:22:14.888 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="raw"
2017-06-02 15:22:14.891 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.663119
2017-06-02 15:22:17.276 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:17.276 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="nat"
2017-06-02 15:22:17.276 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="nat"
2017-06-02 15:22:17.281 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=5.276146
2017-06-02 15:22:18.512 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:18.512 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="filter"
2017-06-02 15:22:18.513 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x4 table="filter"
2017-06-02 15:22:18.518 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=5.927332
2017-06-02 15:22:19.631 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:19.632 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="raw"
2017-06-02 15:22:19.632 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="raw"
2017-06-02 15:22:19.635 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="nat"
2017-06-02 15:22:19.635 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="nat"
2017-06-02 15:22:19.646 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=14.057691
2017-06-02 15:22:19.657 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:19.657 [INFO][82] table.go 632: Invalidating dataplane cache ipVersion=0x6 reason="refresh timer" table="filter"
2017-06-02 15:22:19.657 [INFO][82] table.go 416: Loading current iptables state and checking it is correct. ipVersion=0x6 table="filter"
2017-06-02 15:22:19.659 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=2.040717
2017-06-02 15:22:20.998 [INFO][82] int_dataplane.go 584: Applying dataplane updates
2017-06-02 15:22:20.998 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x4
2017-06-02 15:22:20.998 [INFO][82] route_table.go 141: Queueing a resync of routing table. ipVersion=0x6
2017-06-02 15:22:20.998 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet"
2017-06-02 15:22:20.998 [INFO][82] ipsets.go 214: Asked to resync with the dataplane on next update. family="inet6"
2017-06-02 15:22:20.999 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet"
2017-06-02 15:22:21.000 [INFO][82] ipsets.go 283: Finished resync family="inet" numInconsistenciesFound=0 resyncDuration=1.272027ms
2017-06-02 15:22:21.000 [INFO][82] ipsets.go 242: Resyncing ipsets with dataplane. family="inet6"
2017-06-02 15:22:21.001 [INFO][82] ipsets.go 283: Finished resync family="inet6" numInconsistenciesFound=0 resyncDuration=953.094µs
2017-06-02 15:22:21.002 [INFO][82] int_dataplane.go 599: Finished applying updates to dataplane. msecToApply=3.496159
```

**Comment by alexbrand** [comment]
Have you had a chance to upgrade to the latest version of KET? I am wondering if this may have been fixed in Calico 2.3.0. In any case, my suggestion would be to open an issue on their project for tracking.

**Comment by alexbrand** [comment]
Closing due to inactivity.
